language/stdlib/modules/LibraTransactionPublishingOption.move

```diff
@@ -2,9 +2,10 @@ address 0x1 {
 
 module LibraTransactionPublishingOption {
     use 0x1::Vector;
-    use 0x1::LibraConfig;
+    use 0x1::LibraConfig::{Self, LibraConfig};
```

Comment:
I think general guidance (confirmed by Sam) is to not use unqualified imports any longer. There might be a few exceptions (like `Libra::Libra`), but otherwise use `M::Item` where possible.

Reply:
I think this is fine actually. The general guidance that we have is that structs/resources (i.e., types) are imported unqualified, but that functions and constants are not. The specific recommendations for importing are [here](https://github.com/libra/libra/blob/master/language/stdlib/CONTRIBUTING.md#imports)